packages/contracts/contracts/interfaces/IUnirep.sol

```diff
@@ -68,6 +62,7 @@ interface IUnirep {
     error NullifierAlreadyUsed(uint256 nullilier);
     error AttesterIdNotMatch(uint160 attesterId);
     error OutOfRange();
+    error InvalidField();
```

Comment:
should update documentation
you can run `yarn contracts errorcode` to see the error name and its sigHash